public/images/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/images/mstile-150x150.png"/>
```

Comment:
I'm not very familiar with the mstiles. How does it work that here it's only one of the images referenced, but you actually added 5 different sizes?

Reply:
Yes, you are right. I'll link them.